charts/substrate-faucet/values.yaml

```
@@ -50,6 +50,10 @@ faucet:
     SMF_BACKEND_INJECTED_TYPES: '{}'
     ## @param faucet.config.SMF_BOT_DRIP_AMOUNT Default amount of tokens to send
     ##
+    SMF_BACKEND_FAUCET_BALANCE_CAP: 100
+    ## @param faucet.config.SMF_BACKEND_FAUCET_BALANCE_CAP The bot wil verify this limit before 
```

Please remove the trailing space.

```suggestion
    ## @param faucet.config.SMF_BACKEND_FAUCET_BALANCE_CAP The bot wil verify this limit before
```